src/JITModule.cpp

```diff
@@ -651,36 +661,57 @@ JITModule &make_module(llvm::Module *for_module, Target target,
         one_gpu.set_feature(Target::D3D12Compute, false);
         string module_name;
         switch (runtime_kind) {
-        case OpenCL:
+        case OpenCLDebug:
+            one_gpu.set_feature(Target::Debug);
+            module_name = "debug_";
+        case OpenCL: // fallthrough
```

Comment:
This deliberately-fallthru stuff can cause a headache in some aggressive-warning variants of Clang, unless we annotate with `[[clang::fallthrough]]` or `[[gnu::fallthrough]]`. I'll add a suitable macro in a separate PR.

Reply:
https://github.com/halide/Halide/pull/3211